_pages/community-participation-guidelines.md

```diff
@@ -1,26 +1,26 @@
 ---
-layout: page
+layout: activity
 title:  "Community Participation Guidelines (Draft) Review"
+subtitle: "In like 2-3 hours"
+image: "/assets/img/guidelines-post-header.png"
 permalink: /community-participation-guideline/
 lang: en
 ref: community-participation-guidelines
 parent: activities
 redirect_from: "/experiments/community-participation-guidelines/"
 ---
 
+<div class="col-md-3 pull-right" markdown="1">
 * TOC
```

Comment:
let's get rid of the TOC, and add sections as discussed through Telegram. Also, let's add the "impact and metrics" on the right side (totally fine to not show on mobile)